doc/bus.rst

```diff
@@ -59,6 +59,17 @@ Filtering
 Message filtering can be set up for each bus. Where the interface supports it, this is carried
 out in the hardware or kernel layer - not in Python.
 
+Example defining two filters, one to pass 11-bit ID 0x451, the other to pass 29-bit ID 0xA0000:
```

Comment:
Can we consolidate with https://github.com/hardbyte/python-can/blob/develop/can/bus.py#L330?